src/components/footer/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Container, Sign, TextLink, TextRouterLink, Frame,
+} from './footer.style';
+
+export default function index() {
+  return (
+    <Container>
+      <Frame>
+        <TextLink href="https://profy.dev/employers">profy.dev</TextLink>
+        <Link to="/"><Sign /></Link>
+        <TextRouterLink to="/terms">Terms &amp; Privacy</TextRouterLink>
```

Comment:
You could name these `LeftLink` and `RightLink` for example to be more descriptive

Reply:
Brilliant 🌟